common/Cargo.toml

```diff
@@ -23,6 +22,8 @@ loggers = [
     "hostname",
     "lazy_static",
     "mc-util-build-info",
+    # `mc-util-serial/std` must be enabled since `sentry`, `slog-gelf`, and `slog-json` enable `serde/std`
+    "mc-util-serial/std",
```

Comment:
i would suggest just saying `loggers => std` in this crate

Reply:
Because `failure/std` should also be enabled? Or for a different reason?

Reply:
i just spent a lot of time trying to make these features work in the past, debugging very annoying enclave-not-building problems, and i have even less ability to debug it when these core crates live in a different repo. i have a few rules of thumb, usually, the simplest implication that is correct is what you should use. If any build that uses loggers must have std available (which is true), the loggers should imply the std feature.

Reply:
the more like, piecemeal additions like this people just add arbitrarily to the cargo tomls, the less maintainable all of this becomes.